Dockerfile

```
@@ -1,5 +1,5 @@
 ############# builder
-FROM golang:1.15.3 AS builder
+FROM golang:1.15.7 AS builder
```

```suggestion
FROM eu.gcr.io/gardener-project/3rd/golang/golang:1.15.7 AS builder
```

Please, update also the alpine image. you can use `eu.gcr.io/gardener-project/3rd/alpine:3.12.3`.

---

you probably mean `FROM eu.gcr.io/gardener-project/3rd/golang:1.15.7 AS builder` sure will update it.